### PR TITLE
Print_Status -> Print_Good (And OCD bits 'n bobs)

### PR DESCRIPTION
**Pull request by g0tmi1k**
When there is success or something is added to `report_*`, use `print_good` to make it stand out.

### CODE REVIEW

**Comment by wvu-r7** [comment]
Welcome back, @g0tmi1k. :)

**Comment by sempervictus** [comment]
That right there, is some serious code review.
Thanks as always @g0tmi1k, your tenacity in this matter is bordering on disturbing ;-)

File: modules/post/windows/gather/enum_termserv.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# post/windows/gather/enum_termserv.rb
-
 ##
 # This module requires Metasploit: http://metasploit.com/download
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
 
+# post/windows/gather/enum_termserv.rb
```
Comment (OJ) [inline modules/post/windows/gather/enum_termserv.rb]: Given this is commented out, can it be removed?

File: modules/post/windows/gather/credentials/vnc.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# post/windows/gather/enum_vnc_pw.rb
-
 ##
 # This module requires Metasploit: http://metasploit.com/download
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
 
+# post/windows/gather/enum_vnc_pw.rb
```
Comment (OJ) [inline modules/post/windows/gather/credentials/vnc.rb]: Given this is commented out, can it be removed?

File: modules/exploits/multi/http/rails_dynamic_render_code_exec.rb [hunk 3]
```diff
@@ -1,3 +1,10 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
+
```
Comment (OJ) [inline modules/exploits/multi/http/rails_dynamic_render_code_exec.rb]: Can remove a couple of these newlines?

File: modules/exploits/multi/http/bassmaster_js_injection.rb [hunk 4]
```diff
@@ -1,3 +1,10 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
+
```
Comment (OJ) [inline modules/exploits/multi/http/bassmaster_js_injection.rb]: Can remove a couple of these newlines?

File: modules/exploits/mainframe/ftp/ftp_jcl_creds.rb [hunk 5]
```diff
@@ -1,3 +1,10 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
+
```
Comment (OJ) [inline modules/exploits/mainframe/ftp/ftp_jcl_creds.rb]: Can remove a couple of these newlines?

File: modules/exploits/linux/http/tr064_ntpserver_cmdinject.rb [hunk 6]
```diff
@@ -1,3 +1,10 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
+
```
Comment (OJ) [inline modules/exploits/linux/http/tr064_ntpserver_cmdinject.rb]: Can remove a couple of these newlines?

File: modules/encoders/x86/service.rb [hunk 7]
```diff
@@ -1,3 +1,9 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
```
Comment (OJ) [inline modules/encoders/x86/service.rb]: Can remove a newline.

File: modules/auxiliary/gather/avtech744_dvr_accounts.rb [hunk 8]
```diff
@@ -1,3 +1,10 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+
+
```
Comment (OJ) [inline modules/auxiliary/gather/avtech744_dvr_accounts.rb]: Can remove a couple of these newlines?

**Review by OJ — COMMENTED** [review]
Thanks @g0tmi1k for this! Changes look fine, just a few little things we can adjust.

@bcook-r7 do we have any test suites that allow us to easily test all the modules that were modified?

**Comment by g0tmi1k** [comment]
Missed me @wvu-r7 ? ;)

Thanks @sempervictus :)

If there is anything I can do to help, just shout @OJ 
I've fixed up all the ones that you reported - as well as then apply it to all the other modules I could see.

**Comment by OJ** [comment]
Looking good @g0tmi1k !

**Review by OJ — APPROVED** [review]
👍

**Review by wvu — CHANGES_REQUESTED** [review]
So, I disagree with this commit because it unbalances the whitespace on the other end of the `class` for a number of modules (my own included). Thoughts on how to make it more consistent?

**Comment by g0tmi1k** [comment]
What can I do to help?

**Comment by wvu-r7** [comment]
Just make the whitespace consistent. Don't remove the extra newline after `class` if there is one before the last `end`. Please and thank you. :)

**Comment by g0tmi1k** [comment]
What about removing the last newline after `class` if there is one?

**Comment by wvu-r7** [comment]
That would be more correct, but then I'd have to change my personal style. 1366 changed files is a lot. :-)

**Comment by g0tmi1k** [comment]
How does it look now to you?

**Comment by busterb** [comment]
I'm going to help move this along before it bit-rots. So far haven't seen anything controversial.

**Comment by busterb** [comment]
I made a few additional changes based on the latest master. And as long as we were touching the splat everywhere and making msftidy run on everything, I updated the http://metasploit.com URLs to use https://metasploit.com.

6300758c46 - use https for metaploit.com links
80d18fae6a - update example modules to have zero rubocop violations
6e06d1a8b1 - update rubocop rules for common practice
1d290d2491 - resurrect one print_error/bad conversion for symmetry in eternalblue (print_bad was added recently by popular demand from the author)
8db3f74b81 - fix a broken link

**Comment by g0tmi1k** [comment]
Thanking you @busterb! 🎉

**Comment by busterb** [comment]
now where's that hash rocket guy when you need him...

**Comment by g0tmi1k** [comment]
Where did you last leave him ;)

**Comment by wvu-r7** [comment]
🙇

**Comment by bcoles** [comment]
```

       !
       !
       ^
      /#\
     /###\
    |=###=|
    |# H #|
    |# A #|
    |# S #|
    |# H #|
    |## ##|
    |# R #|
    |# K #|
    |# T #|
    |#####|
   /|##!##|\
  /#|##!##|#\
 /##/##!##\##\
|##/(     )\##|
|#/  #####  \#|
|/  #######  \|
    #######
     #####
      ###
      ###
       #
       #
       #
       #
```

**Pull request merged**